docs/en/index.rst

```diff
@@ -27,7 +27,8 @@ Add the authentication to the middleware. See the CakePHP `documentation
 <http://book.cakephp.org/3.0/en/controllers/middleware.html>`_ on how to use
 middleware if you are not familiar with it.
 
-Example of configuring the authentication middleware using ``authentication`` application hook::
+Example of configuring the authentication middleware using the ``getAuthenticationService``
```

Comment:
() missing if that is a method.

Reply:
Well, I agree, but I assumed we do not add them as ``authentication`` didn't have one, either.

Reply:
I prefer `()` in method names in docs.